frontend/pages/home/components/ICRC/detail/transaction/SendForm/TransactionAmount.tsx

```diff
@@ -50,27 +52,17 @@ export default function TransactionAmount() {
     </>
   );
 
-  async function onChangeAmount(e: ChangeEvent<HTMLInputElement>) {
-    try {
-      const amount = e.target.value.trim();
-      const isValid = Number(amount) >= 0 || amount !== "";
-
-      if (!isValid) {
-        setErrorAction(TransactionValidationErrorsEnum.Values["error.invalid.amount"]);
-        return;
-      }
-      setAmountAction(amount);
+  function onChangeAmount(e: ChangeEvent<HTMLInputElement>) {
+    const amount = e.target.value.trim();
+    setAmountAction(amount);
+    const isValidAmount = validateAmount(amount, Number(sender.asset.decimal));
```

Comment:
is value is "" will get NaN. please verify

Reply:
Verified, if "" the isValidAmount will be still boolean.

Reply:
Verified, if amount is "" will get still boolean.